kotlinx.fuzz.gradle/build.gradle.kts

```diff
@@ -40,7 +40,7 @@ val loggerImplementationProperty = "kotlinx.fuzz.logger.implementation"
  * If changed, consider changing there as well
  */
 tasks.test {
-    val property = System.getProperty(logLevelProperty)
+    val property = System.getProperty(logLevelProperty, "null")
```

Comment:
Why have you added this "null"? 

Also, if there is a good reason, you can replace `?.` with just `.` below

Reply:
It was some kind of brain lag, sorry(